pallets/msa/src/lib.rs

```diff
@@ -905,6 +929,66 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Withdraw all available tokens from the account associated with the MSA, to the account of the caller.
+		///
+		/// The `origin` parameter represents the account from which the function is called and must be the account receiving the tokens.
+		///
+		/// The function requires one signature: `msa_owner_proof`, which serve as proof that the owner of the MSA authorizes the transaction.
+		///
+		/// The necessary information for the withdrawal authorization, the destination account and the MSA ID, are contained in the `authorization_payload` parameter of type [AddKeyData].
+		/// It also contains an expiration block number for the proof, ensuring it is valid and must be greater than the current block.
+		///
+		/// # Events
+		/// * `pallet_balances::Event::<T,I>::Transfer` - Transfer token event
```

Comment:
Anyone know how to make this a doc link correctly (ie, to another pallet)? If I add the brackets to make this a link, it fails linting as broken doc link.